cpu/stm32f1/ldscripts/stm32f103re-slots.c

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Changing the extension led to:

```console
arm-none-eabi-gcc: warning: /Users/facosta/git/RIOT-OS/RIOT/cpu/stm32f1/ldscripts/stm32f103re-slots-template.ld: linker input file unused because linking not done
```